README.md

```diff
@@ -70,6 +70,7 @@ $ cd iceberg-go/cmd/iceberg && go build .
 | Rename Table             |  X   |      |          |  X   |
 | Drop Table               |  X   |      |          |  X   |
 | Alter Table              |      |      |          |      |
+| Check Table Exists       |  X   |      |          |      |
 | Set Table Properties     |      |      |          |      |
```

Comment:
via `UpdateTable` you can set table properties, so we can probably also mark this as supported (for both REST and Glue)

Reply:
Thanks. I have updated the README